python/lsst/summit/utils/utils.py

```diff
@@ -647,7 +644,7 @@ def getSite() -> str:
         return "usdf-k8s"
 
     # we have failed
-    raise ValueError("Location could not be determined")
+    return "UNKOWN"
```

Comment:
Note that butlerUtils and efdUtils each contain a `try..except` block that checks for `ValueError`. Suggest modifying those to remove the `try..except` or just ticketing the change.

Reply:
Thanks. Changed one, and ticketed the one that appears to be fine and I'm too scared to touch right now!